Tasks/HelmDeployV0/src/helmcommands/helmchartsave.ts

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+import tl = require('azure-pipelines-task-lib/task');
+import helmcli from "../helmcli";
+import * as helmutil from "../utils";
+
+/*
+Saves a helm chart to local
+*/
+
+export function addArguments(helmCli: helmcli): void {
+    helmCli.addArgument("save");
+
+    var chartPath = tl.getInput("chartPath", true);
```

Comment:
What if the user gives chartName as the input instead of chartPath?

Reply:
Renamed the input variable to clarify the intent.